### PR TITLE
[SMALLFIX] Fix bug when extensions directory is not present

### PR DESCRIPTION
**Pull request by madanadit**
Create extensions directory if does not exist

### CODE REVIEW

File: core/server/common/src/main/java/alluxio/cli/extensions/command/InstallCommand.java [hunk 1]
```diff
@@ -65,6 +66,10 @@ public String getDescription() {
   public int run(CommandLine cl) {
     String uri = cl.getArgs()[0];
     String extensionsDir = Configuration.get(PropertyKey.EXTENSIONS_DIR);
+    File dir = new File(extensionsDir);
+    if (!dir.exists() && !dir.mkdirs()) {
+      System.err.println("Failed to create extensions directory " + extensionsDir);
```
Comment (aaudiber) [inline core/server/common/src/main/java/alluxio/cli/extensions/command/InstallCommand.java]: return -1?

File: core/server/common/src/main/java/alluxio/cli/extensions/command/InstallCommand.java [hunk 2]
```diff
@@ -65,6 +66,10 @@ public String getDescription() {
   public int run(CommandLine cl) {
     String uri = cl.getArgs()[0];
     String extensionsDir = Configuration.get(PropertyKey.EXTENSIONS_DIR);
+    File dir = new File(extensionsDir);
+    if (!dir.exists() && !dir.mkdirs()) {
```
Comment (aaudiber) [inline core/server/common/src/main/java/alluxio/cli/extensions/command/InstallCommand.java]: this creates the directory locally, but we might install on other hosts - should we also add a remote mkdir before rsyncing to other nodes?

File: core/server/common/src/main/java/alluxio/cli/extensions/command/InstallCommand.java (hunk 1 above)
Comment (madanadit) [reply core/server/common/src/main/java/alluxio/cli/extensions/command/InstallCommand.java]: thx

File: core/server/common/src/main/java/alluxio/cli/extensions/command/InstallCommand.java (hunk 2 above)
Comment (madanadit) [reply core/server/common/src/main/java/alluxio/cli/extensions/command/InstallCommand.java]: rsync will create parent dir

**Comment by madanadit** [comment]
@aaudiber Thanks. Addressed your comments

**Pull request merged**